liveschool.ex

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env elixir
+
+LiveSchool.Cli.main(System.argv())
```

Comment:
I'm not totally sure of the intended usage of this tool but based on this file I'm assuming its a CLI script. Any reason not to make it executable via an escript? I'd also want to see a more robust API for such a CLI script. I'd suggest leveraging [Option Parser](https://hexdocs.pm/elixir/1.12/OptionParser.html) to parse command line arguments rather than grabbing `System.argv` directly.